client/components/Events/index.js

```diff
@@ -25,10 +22,13 @@ class EventPage extends Component {
   // To get the clicked event and a recommended event
   componentDidMount(){
     window.scrollTo(0, 0);
```

Comment:
What does this line do? Are monitoring user scroll activities?

Reply:
The page displays the middle section on load. I can't say why that happens but the line above ensures that the top of the page is what displays on load. I think we'll solve that issue later.